scripts/gen_optillm_dataset.py

```diff
@@ -28,7 +28,7 @@ async def generate_response(prompt: str, **kwargs) -> Dict[str, Any]:
         }
     else:
         # Use OptILM with the specified approach
-        client = AsyncOpenAI(api_key="none", base_url="http://localhost:8000/v1")
+        client = AsyncOpenAI(api_key="none", base_url="http://localhost:8080/v1")
```

Comment:
Doesn't `vllm` run on port 8000 by default? I recall I had to switch the `optillm` proxy port to 8080 due to a conflict like this

Reply:
Yes vllm is on 8000, the proxy runs on 8080 and calls the vllm server on 8000. So here we connect to the proxy, at 8080.